### PR TITLE
Translating quiz into ko(Korean) (in progress)

### PR DESCRIPTION
**Pull request by DaehunGwak**
🥳

### Status
- [x] 1-1
- [x] 1-2
- [x] 1-3
- [ ] 2
- [ ] 2-1
- [ ] 2-2
- [ ] 2-3
- [ ] 2-4
- [ ] 3
- [ ] 3-1
- [ ] 3-2
- [ ] 3-3
- [ ] 4
- [ ] 5
- [ ] 5-1
- [ ] 5-2
- [ ] 5-3
- [ ] 6
- [ ] 6-1
- [ ] 6-2
- [ ] 6-3
- [ ] 6-4
- [ ] 6-5
- [ ] 6-6
- [ ] 7
- [ ] 7-1
- [ ] 7-2
- [ ] 7-3
- [ ] 7-4

### CODE REVIEW

File: 1-getting-started-lessons/3-accessibility/translations/.github/pre-lecture-quiz.ko.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+*수업 들으시면서 퀴즈를 완성해주세요.*
+
+1. 접근가능한 웹사이트는 어떤 툴로 점검할 수 있을까요?
+
+- [ ] Lighthouse
+- [ ] Deckhouse
+- [ ] Cleanhouse
+
+2.. 시각 장애인 유저들을 위한 접근가능성 테스트 용으로 스크린 리더가 필요합니다.
```
Comment (minwook-shin) [inline 1-getting-started-lessons/3-accessibility/translations/.github/pre-lecture-quiz.ko.md:9]: 안녕하세요! 기여 감사드립니다. 한 가지 코멘트를 남깁니다.

"Accessibility" 를  "접근성", "접근가능성"으로 번역하신 것으로 보이는데, 혹시 한 가지로 맞추는 것은 어떨까요?
제 개인적인 의견입니다.
어떤 단어를 사용하시는 지에 따라서는 전적으로 기여하시는 분의 선택으로 존중합니다.

참고로 말씀드리자면, MDN 에서는 "접근성"으로 번역되고 있습니다. ㅎㅎ..
ref : https://developer.mozilla.org/ko/docs/Learn/Accessibility

다시 한 번 감사의 말씀드립니다. 감사합니다.

Comment (DaehunGwak) [reply 1-getting-started-lessons/3-accessibility/translations/.github/pre-lecture-quiz.ko.md:9]: 안녕하세요! 넵 좋은 말씀 감사합니다
"접근성"으로 수정하도록 하겠습니다 👍

**Comment by jlooper** [comment]
hi! thank you for this work. Before you do too much, can you look at our proposal to have Microsoft forms handle the quizzes: https://github.com/microsoft/Web-Dev-For-Beginners/issues/85 - the translation _sort of_ works...let me know what you prefer!

**Comment by DaehunGwak** [comment]
@jlooper Hi!
Thank you for sharing the proposal. 👍 
I think the browser translation is not perfect to me.
That's why I've been translating, so can't this be reflected in the Microsoft form?

@minwook-shin Hello! first, thank you for your feedback :)
I think like above, what do you think? Should I continue with the translation?

**Comment by minwook-shin** [comment]
I agree that's idea.
Browser translation support is not automatic (in my browser case), so I think this is limited support.
But, browser translation support is required because it can support variety of languages.
If translation exists, How about creating separate Microsoft form?
Thank you!

**Comment by jlooper** [comment]
hi, I don't mind if separate Microsoft forms are available and linked; it's quite a job to create them, though (thus I haven't gotten very far :)) - would anyone like to take this one?

**Comment by DaehunGwak** [comment]
@jlooper hello! I'm sorry for checking late.

> would anyone like to take this one?

I don't understand this sentence a little bit...
If there's a series of tasks that move the korean quiz to the microsoft form, and if it's something I can do, can you let me know? I'll try :)

**Comment by jlooper** [comment]
Hi @DaehunGwak Thank you for your help with the translations. I think that if you create a form and open it so that everyone can use it that would be nice. Maybe you can do a sample and we can see how it looks. The trick with forms is that someone will have access to all the student answers unless each student works with their own copy of the form. Are the forms clonable?

**Comment by jlooper** [comment]
hi, a quick note! I've started migrating quizzes to a separate web app. I will get the localizations working there, and we can add the Korean translations to a .json file. I think this is the best way to handle localized quizzes; the forms don't work well for this and I don't want to have sole ownership of them. Better to have everything in the open :).

**Comment by DaehunGwak** [comment]
@jlooper hello! thank you for sharing the quiz app!
this pr is closed, and i'll pr kr translation PR on the quiz app :)

**Pull request closed without merging**